README.md

```diff
@@ -124,8 +122,7 @@ browserify -g [ uglifyify --ignore '**/node_modules/weakmap/*' ] ./index.js
 var bundler = browserify('index.js')
 
 bundler.transform({
-    global: true
-  , ignore: [
+  ignore: [
```

Comment:
ok, so what i am going to do it let this run through uglifyify and then delete this from opts, since uglify-es doesn't accept the `global` option.